python/example_code/rds/lending_library/rds_tools/aurora_tools.py

```diff
@@ -0,0 +1,137 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+Purpose
+
+Shows how to use the AWS SDK for Python (Boto3) to create and delete an Amazon Aurora
+(Aurora) database cluster and create and delete an AWS Secrets Manager secret.
+"""
+
+import json
+import logging
+import sys
+from botocore.exceptions import ClientError
+
+sys.path.append('../../..')
```

Comment:
Just FYI, this is a little magic and may create some oddities if users copy code elsewhere. That may be acceptable for the example though.

Reply:
Added a comment to explain a bit more about what this is for, in case users encounter weirdness.